chapters/en/Unit 1 - Fundamentals/imaging.mdx

```diff
@@ -0,0 +1,29 @@
+
+6. Mathematical Tools in Image Processing
+Mathematical tools are indispensable in digital image processing. Set theory, for instance, is crucial for understanding and performing operations on images, particularly binary images. In these images, pixels are typically categorized as either foreground (1) or background (0). Operations such as union and intersection in set theory are used to determine relationships between features represented by pixel coordinates. Intensity transformations and spatial filtering are other key mathematical tools. They focus on manipulating pixel values within an image, where operators are applied to single images or a set of images for various purposes, like noise reduction.
+
+7. Spatial Filtering Techniques and Image Enhancement
+Spatial filtering encompasses a broad range of applications in image processing, primarily focused on modifying images by altering each pixel's value based on its neighbouring pixels' values. Techniques include linear spatial filters, which can either blur (lowpass filters) or sharpen (high pass filters) an image. The properties and applications of different filter kernels, such as the Gaussian filter and box filters, are contrasted. Sharpening filters, which emphasize transitions in intensity, are often implemented through digital differentiation techniques like the Laplacian, highlighting edges and discontinuities in an image.
+
+8. Image Restoration and Reconstruction Techniques
+Image restoration focuses on objectively recovering a degraded image using knowledge about the degradation phenomenon. It often involves modelling the degradation process and applying inverse processes to regain the original image. In contrast, image enhancement is more subjective, aiming to improve the visual appearance of an image. Restoration techniques include dealing with issues like noise, which can originate from various sources during image acquisition or transmission. Advanced filters, both adaptive and non-adaptive, are explored for their noise-reduction capabilities. In the field of medical imaging, particularly in computed tomography (CT), image reconstruction from projections is a crucial application.
+
+9. Colour in Image Processing
+Colour is a powerful descriptor in image processing, playing a crucial role in object identification and recognition. Colour image processing includes both pseudo-Colour and full-Colour processing. Pseudo-Colour processing assigns Colours to grayscale intensities, while full-Colour processing uses actual Colour data from sensors. Understanding the fundamentals of Colour, including human Colour perception, the Colour spectrum, and the attributes of chromatic light, is key. Different Colour models, such as RGB for monitors and cameras and CMY/CMYK for printing, standardize Colour representation in digital imaging.

```

Comment:
"understanding the fundamentals of Colour, including human Colour perception, the Colour spectrum, and the attributes of chromatic light, is key." I would try to explain this difference here because it is not clear from the text.